ui-core/build.gradle

```diff
@@ -38,6 +42,7 @@ dependencies {
 
     // Dependencies
     implementation libraries.androidx.appcompat
+    api libraries.androidx.constraintlayout
```

Comment:
`PaymentInProgressView` is a `ConstraintLayout` that is exposed to other modules. If modules use this view and we wouldn't use `api` those modules would have to do `implementation libraries.androidx.constraintlayout` themselves. 

Use `api` if the insides of a dependency are publicly exposed. If not, use `implementation`. Always favour `implementation` though, because it speeds up build times quite a bit.